browser/src/control/Control.JSDialog.js

```diff
@@ -859,6 +858,10 @@ window.L.Control.JSDialog = window.L.Control.extend({
 			this.createContainer(instance, documentFragment);
 			this.createDialog(instance);
 
+			if (instance.dialogid && instance.dialogid === "FindReplaceDialog") {
```

Comment:
I'm not happy with hardcoding the id, but no idea now if we could do it better.
As it is some new code. Could we move it to TS so it will be less to convert later?

Please create `browser/src/control/jsdialog/Definitions.DialogModification.ts`
And there a map:   `(dialogId: string)  -> customize_callback(instance): void`
with getter `JSDialog.getDialogModificationCallback(dialogId: string)` which will return found one.

Similar to `menuDefinitions` in `Definitions.Menu.ts`

Then here we can just use:
```
const modifyCallback = JSDialog.getDialogModificationCallback(instance.dialogid);
if (modifyCallback) modifyCallback(instance);
```